app/models/distributions/podcast_distribution.rb

```diff
@@ -23,9 +23,22 @@ def get_podcast
   end
 
   def podcast_attributes
-    {
+    attrs = {
       prx_uri: polymorphic_url(['api', owner], only_path: true),
-      prx_account_uri: api_account_path(account)
+      prx_account_uri: api_account_path(account),
+      published_at: Time.now
```

Comment:
I don't know a better value to set this to, so here goes - just needs to have a value.